srtcore/epoll.cpp

```diff
@@ -719,10 +904,11 @@ int CEPoll::update_events(const SRTSOCKET& uid, std::set<int>& eids, const int e
 #if ENABLE_HEAVY_LOGGING
 static void PrintEpollEvent(ostream& os, int events)
 {
-    static pair<int, string> namemap [] = {
+    static pair<int, const char*> const namemap [] = {
```

Comment:
There was a crash detected in some unique cases referring to this. At least having raw strings in this constant doesn't disturb and prevent any dynamic objects from handling it.